cli/serve.go

```diff
@@ -68,8 +69,8 @@ func runServer(c Config) error {
 		return err
 	}
 
-	resourceService := resource.NewService(resourceRepository, moduleRepository)
-	providerService := provider.NewService(providerRepository)
+	resourceService := resource.NewService(resourceRepository, moduleRepository, time.Now)
+	providerService := provider.NewService(providerRepository, time.Now)
```

Comment:
Something seems off with this conflict resolution you have done. Because this is already in the master.